pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -340,15 +347,12 @@ func (o *Options) Run() error {
 		return nil
 
 	default:
-		if c := findClusterOperatorStatusCondition(cv.Status.Conditions, configv1.OperatorDegraded); c != nil && c.Status == configv1.ConditionTrue {
-			prefix := "No upgrade is possible due to an error"
-			if c := findClusterOperatorStatusCondition(cv.Status.Conditions, configv1.OperatorProgressing); c != nil && c.Status == configv1.ConditionTrue && len(c.Message) > 0 {
-				prefix = c.Message
-			}
-			if len(c.Message) > 0 {
-				return fmt.Errorf("%s:\n\n  Reason: %s\n  Message: %s\n\n", prefix, c.Reason, strings.ReplaceAll(c.Message, "\n", "\n  "))
+		if c := findClusterOperatorStatusCondition(cv.Status.Conditions, clusterStatusFailing); c != nil {
+			if c.Status != configv1.ConditionFalse {
+				fmt.Fprintf(o.Out, "%s=%s:\n\n  Reason: %s\n  Message: %s\n\n", c.Type, c.Status, c.Reason, strings.ReplaceAll(c.Message, "\n", "\n  "))
 			}
-			return fmt.Errorf("The cluster can't be upgraded, see `oc describe clusterversion`")
+		} else {
+			fmt.Fprintf(o.ErrOut, "warning: No current %s info, see `oc describe clusterversion` for more details", clusterStatusFailing)
```

Comment:
Looks like this is a new addition from last time I had reviewed the PR or I had missed it. I am not sure what would be the output for this.

Reply:
not new, this is from f42948c1d0, my original attempt in this PR:

```console
$ git --no-pager show --pretty=fuller f42948c
commit f42948c1d04d2edef347f28c22d0b95611be360c
Author:     W. Trevor King <wking@tremily.us>
AuthorDate: Mon Aug 16 17:29:58 2021 -0700
Commit:     W. Trevor King <wking@tremily.us>
CommitDate: Mon Aug 16 17:29:58 2021 -0700

    pkg/cli/admin/upgrade: Report on Failing!=False conditions
...
-                       return fmt.Errorf("The cluster can't be upgraded, see `oc describe clusterversion`")
+               } else {
+                       fmt.Fprintf(o.ErrOut, "warning: No current %s info, see `oc describe clusterversion` for more details", ClusterStatusFailing)
                }
 
                if c := findCondition(cv.Status.Conditions, configv1.OperatorProgressing); c != nil && len(c.Message) > 0 {
@@ -325,12 +329,12 @@ func (o *Options) Run() error {
                                fmt.Fprintln(o.Out, c.Message)
                        }
                } else {
-                       fmt.Fprintln(o.ErrOut, "warning: No current status info, see `oc describe clusterversion` for more details")
+                       fmt.Fprintf(o.ErrOut, "warning: No current %s info, see `oc describe clusterversion` for more details.\n", configv1.OperatorProgressing)
                }
...
```

The output would be:

    warning: No current Failing info, see `oc describe clusterversion` for more details`

when there was no `Failing` condition.  But I see I'm missing a trailing newline.  Fixed with ea3dd7068 -> 1900577fa.